bin/tasks/build.js

```diff
@@ -37,6 +40,15 @@ export const setSpawnParams = ctx => {
   };
 };
 
+const getOutputDir = buildLog => {
+  const outputIndex = buildLog.lastIndexOf('Output directory: ');
+  if (outputIndex === -1) return undefined;
+  const remainingLog = buildLog.substr(outputIndex + 18);
```

Comment:
What's this magic number 18? Is it the length of `'Output directory: '`?

Reply:
Yep